triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerImpl.scala

```diff
@@ -83,9 +132,14 @@ object TriggerRunnerImpl {
               // is sent to a now terminated actor. We should fix this
               // somehow™.
               ctx.pipeToSelf(trigger) {
-                case Success(_) => Failed(new RuntimeException("Trigger exited unexpectedly"))
-                case Failure(cause) => Failed(cause)
+                case Success(_) =>
+                  Failed(new RuntimeException("Trigger exited unexpectedly"))
+                case Failure(cause) =>
+                  Failed(cause)
               }
+              // Report to the server that this trigger is entering
```

Comment:
If I have a token that gives me `readAs` but not `actAs` the trigger will start up successfully but fail as soon as it tries to do anything. A trigger without `actAs` is completely useless so maybe it would make sense to change the logic for detecting that you only have a single party to verify that the party has `actAs`?

Reply:
That's a good suggestion. I'll implement.